src/app/reporting/Engine.cpp

```diff
@@ -336,6 +363,18 @@ void Engine::Run()
         mCurReadHandlerIdx = (mCurReadHandlerIdx + 1) % CHIP_IM_MAX_NUM_READ_HANDLER;
         readHandler        = imEngine->mReadHandlers + mCurReadHandlerIdx;
     }
+
+    InteractionModelEngine::GetInstance()->ReleaseClusterInfoList(mpChangePathList);
+}
+
+CHIP_ERROR Engine::SetChange(ClusterInfo & aClusterInfo)
```

Comment:
I honestly preferred the term  'dirty' before since it's a well understood term in the industry (especially when it comes to caches, and the concept of invalidation).

Since you've moved the dirty flag out of ClusterInfo, it stands to reason this could still use the term 'dirty'.

Otherwise, if you want to stick to 'change', then this should be past-tense:

```suggestion
CHIP_ERROR Engine::MarkChanged(ClusterInfo & aClusterInfo)
```

Reply:
update it with SetDirty, thanks